data/strings/strings.txt

```diff
@@ -2890,7 +2890,7 @@
     mr = खरेदी
     nb = Shopping
     nl = Winkelen
-    pl = Shopping
+    pl = Zakupy
```

Comment:
Is it present in categories? If no, clicking/typing this word won't find anything.